bin/transform-gisaid

```diff
@@ -153,8 +213,8 @@ if __name__ == '__main__':
             )
         )
 
-    for unused_gisaid_epi_isl in annotations.get_unused_annotations():
-        print(f"WARNING: annotation for {unused_gisaid_epi_isl} was not used.")
+    #for unused_gisaid_epi_isl in annotations.get_unused_annotations():
```

Comment:
Can this commented code be removed if it's replaced by your added functionality in this PR?